docs/5.0/kubernetes-access.md

```diff
@@ -5,61 +5,244 @@ description: How to set up and configure Teleport for Kubernetes access with SSO
 
 # Teleport Kubernetes Access
 
-Teleport has the ability to act as a compliance gateway for managing privileged
-access to Kubernetes clusters. This enables the following capabilities:
+Teleport manages privileged access to Kubernetes clusters.
 
-* Teleport acts as a single authentication endpoint for both SSH and multiple
-  Kubernetes clusters. Users can authenticate against a Teleport Access Plane using
-  Teleport's [`tsh login`](cli-docs.md#tsh-login) command and retrieve credentials
-  for both SSH and Kubernetes API.
-* Users RBAC roles are always synchronized between SSH and Kubernetes, making
-  it easier to implement policies like _developers must not access production
-  data_.
-* Complete `kubectl` auditing and session recordings for `kubectl exec`
-* Multi Kubernetes Support. Login to Teleport once and quickly switch between multiple K8s clusters using [`tsh kube login`](cli-docs.md#tsh-kube-login)
+* Users can login once using [`tsh login`](cli-docs.md#tsh-login) and
+  switch between multiple clusters using [`tsh kube login`](cli-docs.md#tsh-kube-login).
+* Admins can use roles to implement policies like `developers must not access production`.
+* Teleport captures `kubectl` events and session recordings for `kubectl exec`.
 
-## Start Here
+## Deploy Teleport In Kubernetes
 
```

Comment:
I think we went too concise. Let's add one sentence that explains what is about to happen:

Teleport running in Kubernetes provides SSO, auditing and serves as a proxy for Kubernetes API server.